### ISSUE TITLE
Test failure: Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest/UnmanagedCallersOnlyTest.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#35946]
failed in job: [runtime-coreclr jitstress 20200505.1 ](https://dev.azure.com/dnceng/public/_build/results?buildId=632801&view=ms.vss-test-web.build-test-results-tab&runId=19723198&resultId=100068&paneView=debug)

Error message
~~~
Return code: 1
Raw output file: /private/tmp/helix/working/A0920915/w/B18909F4/e/Interop/UnmanagedCallersOnly/Reports/Interop.UnmanagedCallersOnly/UnmanagedCallersOnlyTest/UnmanagedCallersOnlyTest.output.txt
Raw output:
BEGIN EXECUTION
/tmp/helix/working/A0920915/p/corerun UnmanagedCallersOnlyTest.dll ''
Running TestUnmanagedCallersOnlyValid...
Running TestUnmanagedCallersOnlyValid_OnNewNativeThread...
Running TestUnmanagedCallersOnlyValid_PrepareMethod...
Running NegativeTest_NonStaticMethod...
Test Failure: TestLibrary.AssertTestException: Assert.Fail: Expected 'System.InvalidProgramException' to be thrown. 
 at TestLibrary.Assert.HandleFail(String assertionName, String message) in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/Assertion.cs:line 735
 at TestLibrary.Assert.Fail(String message, Object[] args) in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/Assertion.cs:line 722
 at TestLibrary.Assert.Throws[T](Action action, String message, AssertThrowsOptions options) in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Common/CoreCLRTestLibrary/Assertion.cs:line 188
 at Program.NegativeTest_NonStaticMethod() in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest.cs:line 305
 at Program.Main(String[] args) in /Users/runner/runners/2.166.4/work/1/s/src/coreclr/tests/src/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest.cs:line 39
Expected: 100
Actual: 101
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/tmp/helix/working/A0920915/p
> /private/tmp/helix/working/A0920915/w/B18909F4/e/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest/UnmanagedCallersOnlyTest.sh
Expected: True

Actual: False


Stack trace
   at Interop_UnmanagedCallersOnly._UnmanagedCallersOnlyTest_UnmanagedCallersOnlyTest_._UnmanagedCallersOnlyTest_UnmanagedCallersOnlyTest_sh() in /Users/runner/runners/2.166.4/work/1/s/artifacts/tests/coreclr/OSX.x64.Checked/TestWrappers/Interop.UnmanagedCallersOnly/Interop.UnmanagedCallersOnly.XUnitWrapper.cs:line 140
~~~

**Comment by BruceForstall** [comment]
@AaronRobinsonMSFT

**Comment by AaronRobinsonMSFT** [comment]
Already fixed this with #35926
